### PR TITLE
Version 1.5

### LINKED ISSUE #2: [Suggestion] Allow placement on ceiling and floors
Amazing mod! Would be even cooler if you could place the player on ceilings and floors.

### PR DESCRIPTION
**Pull request by theorzr**
Pending SVG support... See #26 

Changes:
- SVG image support
- Floor and ceiling placement with orientation support

### CODE REVIEW

**Comment by samuelh2005** [comment]
SVGs are in #28 now.

File: src/main/java/fr/theorozier/webstreamer/display/DisplayBlock.java
Comment (samuelh2005) [inline src/main/java/fr/theorozier/webstreamer/display/DisplayBlock.java:1]: @mindstorm38 This looks like a massive improvement. I guess we can close #2 once this PR is merged.

Comment (theorzr) [reply src/main/java/fr/theorozier/webstreamer/display/DisplayBlock.java:1]: This is massive yes! The rendering code also have been reworked, maybe to allows sign-post like rotation in the future... I added this to be closed upon merge.

**Comment by theorzr** [comment]
Released: https://modrinth.com/mod/webstreamer/version/1.5.0

**Pull request merged**